### PR TITLE
sda-4206 : add authorizations feature review and self feature review models.

### PR DESCRIPTION
**Pull request by ciaranRoche**
Part of : https://issues.redhat.com/browse/SDA-4206

@tiwillia PTAL we need to get these into the SDK for [SDA-4206](https://issues.redhat.com/browse/SDA-4206)

### CODE REVIEW

File: model/authorizations/v1/feature_review_request_type.model [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (tzvatot) [inline model/authorizations/v1/feature_review_request_type.model:3]: 2021

File: model/authorizations/v1/feature_review_resource.model [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (tzvatot) [inline model/authorizations/v1/feature_review_resource.model:2]: 2021

File: model/authorizations/v1/feature_review_response_type.model [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (tzvatot) [inline model/authorizations/v1/feature_review_response_type.model:2]: 2021

**Comment by Lir10** [comment]
LGTM

**Comment by ciaranRoche** [comment]
@jhernand  can you have a look at this and merge if you are happy with it?

**Comment by tiwillia** [comment]
It looks right to me as well - merging. Juan, if you do review and see any issues please let me know.

**Comment by jhernand** [comment]
There are no references to the new resources from the root resource, or from any other existing resource, so the new methods will not be reachable.

**Pull request merged**